src/Native/build.sh

```diff
@@ -98,3 +98,7 @@ set -x # turn on trace
 cmake "$DIR" -G "Unix Makefiles" $__cmake_defines
 set +x # turn off trace
 make install
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    echo "Changing libMklImports.dylib's executable path within libSymSgdNative.dylib so that loader can find it."
+    install_name_tool "-change" @loader_path/libMklImports.dylib @rpath/libMklImports.dylib "$RootRepo"/bin/x64."$__configuration"/Native/libSymSgdNative.dylib
```

Comment:
>install_name_tool "-change" @loader_path/libMklImports.dylib @rpath/libMklImports.dylib [](start = 4, length = 87)

I'll do this first part directly to the library, after we produce it. We can remove it from the script later.